packages/ComponentLibrary/src/components/Nav/RigthComponents/RightButtons.tsx

```diff
@@ -26,7 +26,7 @@ export default function RightButtons() {
     <>
       <Box style={boxStyles}>
         <IconButtons />
-        <Tooltip title="Account settings">
+        <Tooltip title="Account settings" arrow>
```

Comment:
The labels have to be generic/params

Reply:
This is going to be the Profile modal component in future, I can add a comment to point that because is unnecessary to do it now.

Reply:
I agree with the comment